src/main/java/account/Account.java

```diff
@@ -0,0 +1,45 @@
+package account;
```

Comment:
Can we delete src/main/java/account folder?

Reply:
was removed on my end, for some reason wasn't deleted on github. that has been done.